assets/styl/_modules/_buttons.styl

```diff
@@ -61,17 +61,18 @@
     transition background-color .2s ease
     color $color-base
 
+    &-icon
```

Comment:
`btn-icon` exist yet line 123
yes it's bad named because it's not a child and neither a modifier but they have common modifier (a modifier for different components should be a theme)